webapp/src/ts/selectors/index.ts

```diff
@@ -84,15 +84,20 @@ export const Selectors = {
     return (id) => contactsState.contactsById.has(id);
   }),
   getSelectedContact: createSelector(getContactsState, (contactsState) => contactsState.selected),
-  getLoadingSelectedContactChildren: createSelector(
+  getSelectedContactDoc: createSelector(
     getContactsState,
-    (contactsState) => contactsState.contacts
+    (contactsState) => contactsState.selected?.doc
+  ),
+  getSelectedContactChildren: createSelector(
+    getContactsState,
+    (contactsState) => contactsState.selected?.children
   ),
   getLoadingSelectedContactReports: createSelector(
     getContactsState,
     (contactsState) => contactsState.loadingSelectedReports
   ),
   getContactsLoadingSummary: createSelector(getContactsState, (contactsState) => contactsState.loadingSummary),
+
```

Comment:
Can we remove the commented block at the bottom of the file now?
Or at least the "contacts" parts that have been migrated?

Reply:
Sure! Thanks, fixed